src/utilities/settings/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface Settings {
+  [key: string]: boolean;
```

Comment:
Can this have an explicit shape? Telling consumers "you can toss anything into this object" doesn't feel that helpful when it comes to showing what's available.

Reply:
How much more explicit could you get without limiting flexibility? Or would you see us opening up the type each time we add a new key?

Reply:
Chatted directly about this

Reply:
Context on that chat: I was thinking adding to the type each time you add a new key - doing that feels like a cheap price to pay to ensure we get "isAdafafaf is not a valid key" warnings if we do stuff like `{isAdafafaf} = useSettings();`.

But Tim made a point that we want to be obtuse about this for REASONS for the moment so I'm happy with this.